project.clj

```diff
@@ -6,7 +6,9 @@
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [prismatic/schema "1.1.6"]]
   :profiles {:dev {:dependencies [[org.clojure/math.combinatorics "0.1.3"]
-                                  [org.clojure/data.fressian "0.2.1"]]}
+                                  [org.clojure/data.fressian "0.2.1"]]
+                   :java-source-paths ["src/main/java"
```

Comment:
You don't have to repeat "src/main/java".
`lein` merges collections together if you don't have metadata on them like `:replace` or `:displace`.

Reply:
updated:
https://github.com/cerner/clara-rules/pull/449/commits/dc754f90f170c0ff6b95133443bda1d361a91463